### DISCUSSION TITLE
Do you have any plans to create a component like the MUI Chip?

### DISCUSSION THREAD

**Discussion by yuya-tani** [twilio-labs/paste#3094] [General] [answered]
I'm trying to migrate FlexUI from 1.x to 2.x and  
I'm currently using MUI, but I would like to use Paste on this occasion.  
But Paste doesn't have a chip like MUI.  
Are there any plans to add chip components in the future?

**Comment by loreina** [comment, marked as answer]
Hi @yuya-tani!

Are you referring to [this component](https://mui.com/material-ui/react-chip/)? 

We have [Display Pill Group](https://paste.twilio.design/components/display-pill-group) and [Form Pill Group](https://paste.twilio.design/components/form-pill-group) components that are similar to it dependent on the use case. We also have [guidance](https://paste.twilio.design/components/display-pill-group#when-to-use-a-display-pill-group) on what context to use either.

If you share a screenshot, our team can share some suggestions. Thanks!

**Reply by yuya-tani** [reply]
Hi. Thank you for answering.

As you said, I was pointing to that chip.
There is something called a pill in paste. This certainly matches what I wanted to use.
This I was missing. very sorry.

Let me use this.
Thanks for your help!

**Reply by yuya-tani** [reply]
By the way, it doesn't look very style when I use it. Do you know why?

```
<Box
    display="flex"
    columnGap="space40"
    rowGap="space60"
    flexWrap="wrap"
    alignItems="center"
    justifyContent="center"
    height="2.25rem"
  >
    <DisplayPill>{WorkerGroup.InsideCall.ViewName}</DisplayPill>
</Box>
```
![image](https://user-images.githubusercontent.com/22019718/225178485-df11e13e-3d9c-4516-9ae0-4db6d3fad65b.png)

I don't think the letters above are pills...

**Reply by SiTaggart** [reply]
Hi

Without a reproduction it is pretty difficult for us to see if there would be any problem. It could be version, not wrapping in Theme Provider etc.

That reproduction might also help you debug yourself. We suggest codesandbox as a great tool for reproductions.

**Reply by yuya-tani** [reply]
Hi Simon Taggart.
Thank you for answering.

You're right, it looks like it should have been wrapped in the theme Provider.
I misunderstood that this was done automatically within the Flex UI. sorry.
Thank you for your help.

Best regard.
